mlflow/tracing/__init__.py

```diff
@@ -1,3 +1,12 @@
+from mlflow.tracing.display import IPythonTraceDisplayHandler
 from mlflow.tracing.provider import disable, enable
 
-__all__ = ["disable", "enable"]
+__all__ = ["disable", "enable", "disable_notebook_display", "enable_notebook_display"]
+
+
+def disable_notebook_display():
+    IPythonTraceDisplayHandler.disable()
+
+
+def enable_notebook_display():
```

Comment:
Will you also add docstring in the follow-up?

Reply:
Added! I originally was concerned about publicizing the feature when it's not ready yet, but i think it's probably all going to go into next release anyway so it's not an issue.